coopr-ui/app/features/clusters/controllers/detail-ctrl.js

```diff
@@ -137,6 +153,13 @@ function ($scope, MYSERVICEPICKER_EVENT, CrudFormBase, $state, $modal, $alert, m
 
   /* ----------------------------------------------------------------------- */
 
+  function showLeaseAlert() {
+    $alert({
+      title: 'Lease extension maxed: ',
+      content: 'maximum lease is ' + $scope.leaseMaxHumanized,
```

Comment:
build the content string on the fly, something like `moment($scope.leaseMaxMs)` maybe ?